Photos/app/src/main/res/layout/feed_fragment.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent" android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/feed_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/action_feed"
+        android:textColor="@android:color/black"
+        android:layout_centerHorizontal="true"
+        android:textSize="20pt"/>
+
+    <GridView
+        android:paddingTop="30dp"
+        android:id="@+id/feed_image_grid"
+        android:layout_below="@id/feed_title"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:numColumns="auto_fit"
+        android:columnWidth="90dp"
+        android:verticalSpacing="10dp"
+        android:horizontalSpacing="10dp"
+        android:stretchMode="columnWidth"
+        android:gravity="center"
+        android:focusable="false"
+        android:focusableInTouchMode="false" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:weightSum="1">
+
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/feed_button_back"
+            android:text="@string/back_arrow"
+            android:textSize="20pt"
+            android:gravity="start|bottom"
+            android:background="@android:color/transparent"
+            android:layout_weight="0.5"
+            />
+
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="0.5"
+            android:textSize="20pt"
+            android:id="@+id/feed_button_next"
+            android:text="@string/next_arrow"
+            android:gravity="end|bottom"
+            android:background="@android:color/transparent"
+            />
```

Comment:
consider making a style for more than one component (like these 2 buttons) that have a lot of the same styling. it will save time and effort.